website/components/plugin-tier-label/style.module.css

```diff
@@ -12,7 +12,9 @@ to match Terraform Registry tier labels.
   display: inline-flex;
   margin: 0;
   padding: 1px 8px 2px 8px;
-  vertical-align: baseline;
+  position: relative;
+  top: -1px;
+  vertical-align: bottom;
```

Comment:
This fixes a minor issue with alignment of labels. Previously, when using labels inline in paragraph text, there were alignment issues relative to text and between consecutive labels. Here's some sample content after the fix:

<img width="743" alt="CleanShot 2021-03-09 at 16 06 55@2x" src="https://user-images.githubusercontent.com/4624598/110541624-02125c80-80f6-11eb-8e9a-3b1ed16232c6.png">